### DISCUSSION TITLE
next.js  renders blank page or partially rendered page

### DISCUSSION THREAD

**Discussion by ravitdx** [vercel/next.js#13744] [Help]
Hi,
I am using nextjs for my website dynamic pages. I am using Apollo graphql to fetch data. Everything works correctly. But, occasionally, for some people frequently, sees a blank page  or partially rendered page or non-interact able page. Is this expected behavior? Can I do something to prevent this?

Thanks.

**Comment by jamesmosier** [comment]
That doesn't sound expected. If you have any code/examples to share that'd be helpful though!
